cli/src/main/java/com/box/l10n/mojito/cli/command/ExtractionDiffCommand.java

```diff
@@ -2,46 +2,54 @@
 
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
+import com.box.l10n.mojito.cli.command.extraction.AssetExtractionDiff;
 import com.box.l10n.mojito.cli.command.extraction.ExtractionDiffService;
+import com.box.l10n.mojito.cli.command.extraction.ExtractionDiffsPaths;
 import com.box.l10n.mojito.cli.command.extraction.ExtractionsPaths;
 import com.box.l10n.mojito.cli.command.extraction.MissingExtractionDirectoryExcpetion;
-import com.box.l10n.mojito.cli.command.param.Param;
 import com.box.l10n.mojito.cli.console.ConsoleWriter;
+import com.box.l10n.mojito.json.ObjectMapper;
+import com.box.l10n.mojito.rest.entity.Repository;
+import com.box.l10n.mojito.rest.entity.SourceAsset;
 import org.fusesource.jansi.Ansi;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Scope;
 import org.springframework.stereotype.Component;
 
+import java.nio.file.Path;
+import java.util.stream.Stream;
+
 /**
  * Computes a diff between to local extractions.
- *
+ * <p>
```

Comment:
@jeeyi FYI, this is a new command that is handy to process pull-request/diffs without calling the server at all. Eventually, if there are new translation units it can sync the delta with the server

Reply:
This is super helpful. Thank you @aurambaj!